test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1268,6 +1269,27 @@ UCS_TEST_P(select_ep_transport, fp8_modify_result_v2)
     verify_symmetric_tl_selection("65", "55");
 }
 
+UCS_TEST_P(select_ep_transport, max_lanes, "IB_NUM_PATHS?=8")
+{
+    modify_config("MAX_RNDV_LANES", "8");
+    modify_config("RNDV_THRESH", "0");
+
+    test_ucp_wireup::init();
+
+    sender().connect(&receiver(), get_ep_params());
+    send_recv(sender().ep(), receiver().worker(), receiver().ep(), 1024, 10);
+
+    for (int i = 0; i < ucp_ep_num_lanes(sender().ep()); ++i) {
+        uct_base_ep_t *base = ucs_derived_of(ucp_ep_get_lane(sender().ep(), i),
+                                             uct_base_ep_t);
+        uint64_t bytes_sent =
+                UCS_STATS_GET_COUNTER(base->stats, UCT_EP_STAT_BYTES_SHORT) +
+                UCS_STATS_GET_COUNTER(base->stats, UCT_EP_STAT_BYTES_BCOPY) +
+                UCS_STATS_GET_COUNTER(base->stats, UCT_EP_STAT_BYTES_ZCOPY);
+        ASSERT_GE(bytes_sent, 0);
```

Comment:
maybe send larger message and expect more than X bytes?
in case the lane sent other data (like wireup)

Reply:
yes, planned to do so (test was commited by mistake)